search-enterprise/latest/using/unsupported.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Unsupported features
+---
+
+The following features, which were supported with Search and Insight Engine 2.x and Search Services 2.x (Solr) are not supported in the latest release for Search Enterprise 3.x (Elasticsearch).
+
+## Index and re-index
+
+* Re-index of permissions and content associated with indexed nodes.
+* Indexing of nodes created during content repository bootstrap. For example, the sample site data. @Engineering what is this example?
+
+## Search features
+
+* Site queries
+* Path queries
+* Aspect queries (works using only the exact aspect name)
+* Tag queries
+* Highlighting
+* Fingerprinting
+* Multilingual support (documents will be accepted and searchable with multiple languages, but only English grammar rules will be applied)
+* Template search
+* Resource limiting
+* Scoped search
+* Statistics
+* Search for optional, mandatory, and excluded elements of a query
+* Search for an exact term
+* Fuzzy matching
+* Grouped search queries
+* Proximity search
+* Search for spans and positions
+* Search using date math
+* Search for boosts
+* Wildcards in phrase queries
+* Exact Match Queries
+* Field Facets Pagination
+* Field Facets Tags Exclusion
+
+## Field queries
+
+* PATH
+* PATHWITHREPEATS
+* PNAME
+* NPATH
+* PARENT
+* PRIMARYPARENT
+* QNAME
+* TAG
+* SITE
+* ANCESTOR
+* PRIMARYASSOCQNAME
+* PRIMARYASSOCTYPEQNAME
+* ISNULL
+* ISNOTNULL
+* ISUNSET
+* FINGERPRINT
+* ISROOT
+* ISCONTAINER
+* ASPECT
+* EXACTASPECT
+* CASCADETX
+* DBID
+* TX
+* TXID
+* INTXID
+* TXCOMMITTIME
+* ACLID
+* INACLTXID
+* ACLTXID
+* ACLTXCOMMITTIME
+* TENANT
+* OWNERSET
+* READERSET
```

Comment:
Remove line (this was never supported for v2)

Reply:
@mjstanford Do you mean READERSET ? 
If it was never supported it is already on the unsupported page?
cheers